microsoft-integration/media-gallery-tool/src/main/webapp/WEB-INF/templates/fragments/common.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<body>
+<head th:fragment="head">
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<link media="all" href="/library/skin/tool_base.css" rel="stylesheet" type="text/css" />
+	<link href="/library/skin/morpheus-default/tool.css" type="text/css" rel="stylesheet" media="all" />
+	<script src="/library/js/headscripts.js" type="text/javascript"></script>
+	<script th:insert="js/common.js"></script>
+	<title th:text="#{index_title}">Microsoft Video Gallery</title>
+</head>
+
+<div th:fragment="bootstrap">
+	<script>includeLatestJQuery('microsoft-video-gallery');</script>
+	<script th:inline="javascript">
+		(function() {
+			const toolId = /*[[${#httpServletRequest.getAttribute('sakai.tool.placement.id')}]]*/'toolId';
```

Comment:
Remove?

Reply:
This is a thymeleaf fallback

Reply:
Hi @mpellicer !! here it is!

index_title=Microsoft Media Gallery

loading=Kargatzen...
no_synchronized=Sinkronizazio aktiborik ez dago gune honetan

tree_view=Zuhaitza gisako bista
tree_view_on=Zuhaitza gisako bista gaitua
tree_view_off=Zuhaitza gisako bista desgaitua
container_empty=Ez da aurkitu itemik

refresh=Eguneratu
search=Bilatu

order_by=Ordenatu honen arabera:
sort_by_name_ascending=Izena (A-Z)
sort_by_name_descending=Izena (Z-A)
sort_by_date_ascending=Data (Berriena hasieran)
sort_by_date_descending=Data (Zaharrena hasieran)

play=Play
info=Erakutsi informazioa

#Info
name=Izena: 
created=Sortuta:
modified=Aldatuta:
path=Bidea:
item_empty=Ez da aurkitu itemik
back=Atzera

#sections
USER=Erabiltzailearen galeria
SHARED=Partekatutako galeria

#delegated access
set_account= Hasi saioa zure Microsoft-eko kontuarekin
configure=Microsoft Login
revoke=Microsoft Logout

#date format
date.format=yyyy/MM/dd HH\:mm

#errors
error.thumbnail=Errorea miniatura berreskuratzean
error.link=Errorea esteka berreskuratzean
error.info=Errorea informazioa berreskuratzean
error.items=Errorea itemak berreskuratzean

#Errors - from exceptions
error.no_credentials_provided=Ez dira kredentzialak eman
error.invalid_credentials_provided=Emandako kredentzialak ez dira onak
error.invalid_token_provided=Emandako token-a ez da ona

Reply:
Thank you!